InterviewBit_problems/Sorting/Hotel Booking/solution.py

```diff
@@ -0,0 +1,29 @@
+'''
+- A hotel manager has to process N advance bookings of rooms for the next season. 
+His hotel has C rooms. Bookings contain a list A of arrival date and a list B of departure date. 
+He wants to find out whether there are enough rooms in the hotel to satisfy the demand.
+
+- Creating a function hotel which will take 3 arguments as arrive, depart and K
+- First we will sort both the list arrive and depart
+- Then we will traverse the list arrive and check for the index elements at i+K that the element in
+arrive is less than depart i.e. arrive date is less than depart date
+- If arrive date is less than depart date then there are no rooms and returning False
+- But if no arrive date is less than depart date then there are enough rooms for N bookings and hence returning True
+'''
+def hotel(arrive, depart, K):
+        arrive.sort()
+        depart.sort()
+        print(arrive,"\n",depart)
```

Comment:
please remove/comment `print(arrive,"\n",depart)` (line 16) since this causes unnecessary output

Reply:
Removed the print statement and the final code is ready